server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1462,6 +1472,10 @@ public Node start() throws NodeValidationException {
         if (remoteClusterStateService != null) {
             remoteClusterStateService.start();
         }
+        final RemoteIndexPathUploader indexCreationListener = injector.getInstance(RemoteIndexPathUploader.class);
```

Comment:
nit: Why not `remoteIndexPathUploader`?

Reply:
used this name earlier and received a feedback for changing the name. This, however, got missed.